<!DOCTYPE><html>

<head>
<title>FYP Supplementary File</title>
<link rel="stylesheet" href="FYP_SF.css">
</head>

<body>

<div class="wrapper" id="title">
    <h1><p>FYP TITLE: MARINE NATURAL PRODUCTS AS POTENTIAL INHIBITORS FOR <i>Streptococcus agalactiae</i> PHOSPHOPENTOMUTASE: A MOLECULAR DOCKING STUDY</p></h1>
</div>
<br><br><br>

<div id="file">
    <h2><p><u>List of supplementary files</u></p></h2>
</div>

<nav class="navbar">
<ul>
    <li><a href="#phase1">Phase 1</a></li>
    <li><a href="#phase2">Phase 2</a></li>
    <li><a href="#phase3">Phase 3</a></li>
</ul>
</nav>
<br><br><br>

<div id="phase1">
    <h2><p><u>Phase 1: Sequence-Based Analysis</u></p></h2>
</div>

<div>
    <h2><p><u>1. Multiple Sequence Alignment</u></p></h2>
    <img height="800px" src="Multiple Sequence Alignment.png">
    <h3><p>Figure 4.2. Illustration of MSA between <i>S. agalactiae</i> phosphopentomutase (Q8CMH7) and its three templates (3M7V, 3M8Z, 3UN2) performed by using Clustal Omega server and visualized by using ESpript 3 server. The black stars indicate all the substrate binding sites of <i>S. agalactiae</i> phosphopentomutase whereas the green stars indicate all the cofactor binding sites of two Mn ions. The yellow bar shows the location of the metalloenzyme domain of the bacterial protein.</p></h3>
</div>

<div>
    <h2><p><u>2. Physicochemical Characterisation</u></p></h2>
    <img height="600px" src="physicochemical characterization table.png">
</div>
<br><br><br><br>

<div id="phase2">
    <h2><p><u>Phase 2: Structure-Based Analysis</u></p></h2>
</div>

<div>
    <h2><p><u>1. Protein Model Construction</u></p></h2>
    <img height="600px" src="Protein Model Construction.jpeg">
    <h3><p>Figure 4.3. The best protein model of phosphopentomutase enzyme predicted by AlphaFold2 with the reference percentage of model confidence per residue being displayed at the bottom.</p></h3>
</div>

<div id="secondary">
    <h2><p><u>2. Topology & Secondary Structure Analysis</u></p></h2>
    <h3><p>Figure 4.4. The secondary structure of phosphopentomutase enzyme generated by PDBSum</p></h3>
    <img height="600px" src="secondary structure.gif">
    <h3><p>Figure 4.5. The motif of phosphopentomutase enzyme.</p></h3>
    <img height="600px" src="secondary structure3.gif">
</div>

<div>
    <h2><p><u>3. Model Evaluation</u></p></h2>
</div>

<div id="evaluation">
    <img height="600px" src="Ramachandran plot (PROCHECK).jpg">
    <h3><p>Figure 4.6. The Ramachandran plot of the modelled phosphopentomutase enzyme (UniProt ID: Q8CMH7) that was generated by PROCHECK test.</p></h3>
    <img height="600px" src="VERIFY3D results.png">
    <h3><p>Figure 4.7. The graph of VERIFY3D results for the predicted phosphopentomutase enzyme.</p></h3>
    <img height="500px" src="ERRAT results.png">
    <img height="500px" src="ERRAT results 2.png">
    <h3><p>Figure 4.8. The bar graph of ERRAT test result for the predicted phoshopentomutase enzyme.</p></h3>
    <img height="400px" src="overall summary report of WHATCHECK result.png">
    <br><br><br>
    <img height="600px" src="ProSA-web results.png">
    <h3><p>Figure 4.9. Z-score results of the predicted phosphopentomutas enzyme analyzed by ProSA-web server</p></h3>
    <img height="600px" src="overall summary report of model evaluation .png">
</div>
<br><br><br>

<div id="phase3">
    <h2><p><u>Phase 3: Protein-Ligand Interaction</u></p></h2>
</div>

<div>
    <h2><p><u>1. Location and Binding of Mn2+ Ions</u></p></h2>
    <img height="400px" src="Location and binding of Mn ions.png">
    <h3><p>Figure 4.10. A complex of phosphopentomutase enzyme that are bound with two ions (Mn-1 and Mn-2) was visualized by using Pymol in the form of both cartoon and stick representation.</p></h3>
    <h2><p><u>2. Location and Binding of Ligand (MNPs)</u></p></h2>
    <img height="500px" src="Location and binding of Ligand (MNP).png">
    <h3><p>Figure 4.11. 15 different active site residues of S. agalactiae phosphopentomutase, which are namely THR-92, SER-140, GLY-141, TYR-161, ASP-165, GLN-169, ARG-202, ILE-204, ARG-206, ARG-217, LYS-249, ASP-298, HIS-303, HIS-340 and HIS-351.</p></h3>
    <h2><p><u>3. Molecular Docking Between S. agalactiae Phosphopentomutase & MNPs</u></p></h2>
    <img height="600px" src="Binding energy values of the 8 MNPs.png">
    <h2><p><u>4. Pharmacokinetic Assessment of MNPs</u></p></h2>
    <img height="500px" src="List of pharmacokinetic properties.png">
    <h2><p><u>5. Visualization of Receptor-Ligand Interaction</u></p></h2>
    <img height="1000px" src="2D Interaction.png">
    <h3><p>Figure 4.12. Visualization of 2D interaction between (a) CMNPD30311, (b) CMNPD30314, (c) CMNPD30308, (d) CMNPD30312, (e) CMNPD30307, (f) CMNPD30310, (g) CMNPD30313, (h) CMNPD30309 and S. agalactiae phosphopentomutase by using Biovia Discovery Studio visualizer. Key amino acid residues that interact with MNPs are represented in circles. Grey colour highlights the MNPs ligand; green colour dotted line indicates conventional hydrogen bonds; medium green colour dotted line indicates van der Waals interactions; light green colour dotted line indicates carbon-hydrogen bonds and light purple colour indicates both alkyl and pi-alkyl interactions. Green colour amino acids represent conventional hydrogen bond interactions; light green colour amino acids without interactions represent van der Waals interactions and the red colour amino acids indicate unfavourable interaction with the formation of unfavourable bumps. The blue halo that surrounds the interacting residues represents the solvent-accessible surface that is proportional to its diameter.</p></h3>
    <h2><p><u>6. Virtual Screening and Selection of MNPs</u></p></h2>
    <img height="550px" src="List of top 3 best MNPs.png"> 
    <br><br><br>
    <img height="1000px" src="Virtual Screening and Selection of MNPs.png">
    <h3><p>Figure 4.13. List of surface 3D and 2D representations of S. agalactiae phosphopentomutase that are bound with top three best MNPs, which are (a) CMNPD30307, (b) CMNPD30309, (c) CMNPD30310 with their respective 2D interactions.</p></h3>
</div>

<footer>
    <h1><p>THANK YOU &#128522!</p></h1>
    <img height="500px" src="thank you protein.gif">
</footer>
<br><br><br><br>

<hr>

<div>
<h4><p>For more information, please contact via the following details:</p></h4>
<a href="mailto: s63713@ocean.umt.edu.my"><h4><p>&#9993 s63713@ocean.umt.edu.my</p></h4></a>
<a href="tel:+6012-8517668">&#9743 +6012-8517668</a>
</div>

</body>

</html>